lropt/robust_problem.py

```diff
@@ -23,6 +24,69 @@
 from lropt.uncertain_canon.uncertain_chain import UncertainChain
 from lropt.uncertainty_sets.mro import MRO
 
+#Different types of MRO:
+#no_mro (No MRO)
+#diff_A_uninit (Different A for each k and uninitialized)
+#diff_A_init (Different A for each k, initialized with a different matrix for each k)
+#same_A (same A for each k)
+MRO_CASE = Enum("MRO_CASE", "no_mro diff_A_uninit diff_A_init same_A")
```

Comment:
This is a great way to simplify it. However, I wonder if we should simplify the entire setting with `MRO` and then two options for different or same matrices per sample. Definitely needs a deeper discussion.